### PR TITLE
move the status bar from the left side to the right

### PR DESCRIPTION
**Pull request by devoncarew**
Move the status line contribution from the left hand side to the right.

screen shot 2015-06-26 at 11 06 28 am

### CODE REVIEW

**Comment by k2b6s9j** [comment]
I'd rather keep the buttons on the left.

**Comment by steelbrain** [comment]
@k2b6s9j Me too

**Comment by k2b6s9j** [comment]
The `No Errors` is good on the right though.

**Comment by steelbrain** [comment]
@k2b6s9j Yep I agree, but for some reasons AsaAyers doesn't, that's why I abandoned https://github.com/AtomLinter/Linter/pull/576

**Comment by devoncarew** [comment]
I actually don't have strong opinions about which side it should be on. I do think the status line contribution should be a little more compact...

**Comment by steelbrain** [comment]
We are happy to merge it in if you keep the tabs on the left and move the status icon to the right. Just like the in the old PR I referenced above.

**Comment by devoncarew** [comment]
Cool, here it is:

screen shot 2015-06-26 at 11 39 49 am

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 * Rename the status line summary (from `Errors` to `Issues`)
 * Rename the `Current File` tab to just `File`
+* Move the `No Issues` status line contribution to the right side of the status line
```
Comment (steelbrain) [inline CHANGELOG.md]: How about
```
Move the `No Issues` badge to the right side of bottom bar
```

**Comment by k2b6s9j** [comment]
:shipit:

**Pull request merged**